bindings/python/parsers/python/model.cpp

```diff
@@ -68,11 +69,38 @@ namespace pinocchio
         std::cout << " degrees of freedom." << std::endl;
       }
 
-      // close interpreter
+      // close the interpreter
+      // cf. https://github.com/numpy/numpy/issues/8097
+#if PY_MAJOR_VERSION < 3
       Py_Finalize();
+#else
+
+      PyObject * poMainModule = PyImport_AddModule("__main__");
```

Comment:
Maybe you should add a ref for those lines to help people in the future for a diagnosis of this code.

Reply:
It is on the new line 73: 
`       // cf. https://github.com/numpy/numpy/issues/8097`

Reply:
Ok. I did not see it. Thanks!